src/firebase/remoteConfig/fetchRemoteConfig.ts

```diff
@@ -10,6 +11,14 @@ export function getIsDemoLoginEnabled(): boolean {
   return remoteConfig().getBoolean('isDemoLoginEnabled');
 }
 
+export function getFeatureFlags(): FeatureFlags | undefined {
+  const featureFlags = remoteConfig().getString('featureFlags');
+  if (featureFlags) {
+    return JSON.parse(featureFlags);
+  }
+  return undefined;
```

Comment:
can this be a default value rather than undefined ?

Reply:
No because we only want one set of fallbacks in the app for if it isn't in the remote config. The undefined behaviour here is intentional